unit/ctest_dg_rad_gyrokinetic.c

```diff
@@ -266,13 +266,15 @@ test_1x(int poly_order, bool use_gpu, double te)
     &confBasis, &basis, poly_order+1, gvm, use_gpu);
 
   // If on GPUs, need to copy n, udrift, and vt^2 onto device
-  struct gkyl_array *prim_moms_dev, *m0_dev;
+  struct gkyl_array *prim_moms_dev, *m0_dev, *vtsq_dev;
```

Comment:
this unit test has a less than ideal naming convention. In general array should be named without _dev, and only those arrays that require host-specific operations (e.g. I/O) should have a _ho or _host twin. If possible change this around. I know you probably looked at some older unit test that gave this impression, but we should use a better practice moving forward, rather than duplicating less than ideal practices.

Reply:
Are you requesting just a change in the names here? If so, is there a convention to stay to other than avoiding the _dev postfix?

Reply:
The best convention is to not use a suffix in the main arrays, regardless of whether they are on the host or the device (per `use_gpu`), and only allocate some `_ho` arrays for projecting ICs, I/O (if needed), and checking results. See this relatively simple test: https://github.com/ammarhakim/gkylzero/blob/fc692f6a30752b36b9be74a0494438d2d9c23d0a/unit/ctest_gyrokinetic_pol_density.c#L50